modules/oadp-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,335 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="migration-oadp-release-notes-1-3-0_{context}"]
+= OADP 1.3.0 release notes
+
+The OpenShift API for Data Protection (OADP) 1.3.0 release notes lists any new features, resolved issues and bugs, and known issues.
+
+[id="new-features-1-3-0_{context}"]
+== New features
+
+.Velero Built-in DataMover
+
+OADP 1.3 includes a Built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses Kopia as the uploader mechanism to read the snapshot data and write to the Unified Repository.
+
+
+:FeatureName: Velero Built-in DataMover
+include::snippets/technology-preview.adoc[]
+
+.Backing up applications with File System Backup: Kopia or Restic
+
+Velero’s File System Backup (FSB) supports two backup libraries, the Restic path and the Kopia path.
+
+Velero allows users to select between the two paths.
+
+For backup, specify the path during the installation through the uploader-type flag, the valid value is either `restic` or `kopia`, or defaults to Kopia if the value is not specified. The selection cannot be changed after the installation.
+
+.GCP Cloud authentication
+
+Allows a user to use Google’s short-lived credentials.
+GCP With Identity Federation allows you to use Identity and Access Management (IAM) to grant external identities IAM roles, including the ability to impersonate service accounts. This eliminates the maintenance and security risks associated with service account keys.
+
+.AWS ROSA STS authentication
+
+You can use OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to back up and restore application data.
+
+ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to speed up the building and delivering of differentiating experiences to your customers.
+
+You subscribe to the service directly from your AWS account.
+
+After the clusters are created, you can operate your clusters with the OpenShift web console or through `cluster-manager-first`. The ROSA service also uses OpenShift APIs and command-line interface (CLI) tools.
```

Comment:
`cluster-manager-first`
What is this?

Reply:
@kaovilai 
I don't know. This is a snippet I received.
Should I remove `cluster-manager-first`?

Reply:
I don't know either. Let's wait for others.

Reply:
@kaovilai 
I can wait till tomorrow morning Israel time, because I need to send this for OCP review tomorrow morning my time. I can always remove it and add it later if necessary.

Reply:
I removed it so I can send it for review. I'm only going to ask them to merge it next week. If you think we should include it, I will add it after the  review before it's merged.